src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -1117,12 +1121,20 @@ public final class Address implements ApiMessage {
       return this;
     }
 
-    public List<String> getUsers() {
+    public List<String> getUsersList() {
       return users;
     }
 
     public Builder addAllUsers(List<String> users) {
-      this.users = users;
+      if (this.users == null) {
+        this.users = new LinkedList<>();
+      }
+      this.users.addAll(users);
+      return this;
```

Comment:
I think it would be better to use an `ArrayList` with a capacity initialized to the size of `users`

Reply:
Done. I'm going to assume the user knows not to call this method on a null param.